nightfall-client/src/services/commitment-storage.mjs

```diff
@@ -688,3 +688,67 @@ export async function findUsableCommitmentsMutex(
     findUsableCommitments(compressedZkpPublicKey, ercAddress, tokenId, _value, onlyOne),
   );
 }
+
+/**
+ * @function getCommitmentsByCompressedPkdList do the role of a service taking care of the
```

Comment:
I prefer `getCommitmentsByCompressedPkdList` instead of getAllCommitmentsByCompressedZkpPublicKeyList, L701 ;p

Reply:
> I prefer `getCommitmentsByCompressedPkdList` instead of getAllCommitmentsByCompressedZkpPublicKeyList, L701 ;p

Cool! I would do this as well!

Reply:
But the name change, is not more compressedPkd.